GalaxyZooTouchTable/GalaxyZooTouchTable/UserConsole.xaml.cs

```diff
@@ -10,19 +14,45 @@ namespace GalaxyZooTouchTable
     public partial class UserConsole : UserControl
     {
         ClassificationPanel Classifier { get; set; }
+        public bool ClassifierOpen { get => classifierOpen; set => classifierOpen = value; }
+
+        private bool classifierOpen = false;
 
         public UserConsole()
         {
             InitializeComponent();
         }
 
         private void StartButton_TouchUp(object sender, TouchEventArgs e)
+        {
+            ToggleClassifier();
+        }
+
+        public void ToggleClassifier()
+        {
+            MoveButton();
+            AddClassifier();
```

Comment:
Is this right? You add a classifier every time you toggle the classifier?

Reply:
Yes. My thought here is that a classifier will be removed when a session ends and a new classifier will introduce a new session. However, this could change as I start to add functionality to the classifier.